velox/exec/RowContainer.h

```diff
@@ -525,6 +525,10 @@ class RowContainer {
         stringAllocator_.freeSpace());
   }
 
+  // Returns the average size of rows in bytes stored in this container.
+  // Returns -1 if the container is empty.
+  int64_t estimatedRowSize() const;
```

Comment:
It would be nice to add a test for this logic to velox/exec/tests/RowContainerTest.cpp

Reply:
Added one unit test for it.